<!--#######################################-->
<!-- Copyright (c) 2023  Dong Zhaorui.     -->
<!-- All rights reserved.                  -->
<!-- Author: Dong Zhaorui 847235539@qq.com -->
<!-- Date  : 2023-11-21                    -->
<!--#######################################-->

<launch> 
    <node name="hex_example" pkg="hex_example" type="hex_example" output="screen" >
        <!-- parameter -->
        <rosparam file="$(find hex_example)/config/ros1/hex_example.yaml"/>
        <param name="max_count" value="20"/>
        <!-- subscribe -->
        <remap from="/in_string" to="/in" />
        <!-- publish -->
        <remap from="/out_string" to="/out" />
    </node>    
</launch>